testing/fixture_test.h

```diff
@@ -20,6 +20,7 @@ namespace testing {
 class FixtureTest : public ThreadTest {
  public:
   FixtureTest();
+  FixtureTest(std::string kernel_file);
```

Comment:
Please add a doc comment about when/how to use this.

Also, single argument ctor should be marked explicit (https://google.github.io/styleguide/cppguide.html#Implicit_Conversions).

Reply:
Done.